CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## How to contribute to guides.rubygems.org
+
+#### **If you want to contribute to RubyGems itself, visit [the RubyGems Guides](http://guides.rubygems.org/contributing/)**
+This guide just covers the details of contributing to the guides themselves.
+
+### Running guides.rubygems.org
+
+The [guides.rubygems.org site](http://guides.rubygems.org) site is built on [Jekyll](https://github.com/jekyll/jekyll).  To run the site locally, you'll first need to install Jekyll via `gem install jekyll`.  Once installed, you can start the development server via `jekyll server` and point your browser to [http://localhost:4000](http://localhost:4000)
```

Comment:
I would just say `The guides are built on..`, we don't need to link to the production site again.